WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -362,22 +365,48 @@ class AztecPostViewController: UIViewController {
 
 // MARK: - Actions
 extension AztecPostViewController {
-    @IBAction func cancelEditingAction() {
+
+    @IBAction func closeWasPressed() {
         cancelEditing()
     }
 
-    @IBAction func displayBlogSelector() {
+    @IBAction func blogPickerWasPressed() {
+        guard isSingleSiteMode == false else {
```

Comment:
@astralbodies - I think this could be the time to question a lot of assumptions from the hybrid editor, so in trying to push this forward here are my thoughts:

Should it ever be possible to select the block picker in single site mode?  This looks way wrong as it is.

Also, why cancel editing?  That feels like a line that should be removed indeed.

I'd consider replacing the `cancelEditing()` line with an assertion failure (unless I'm missing something else).